components/ClassDisplay.tsx

```diff
@@ -15,7 +15,7 @@ const ClassDisplay: React.FC<ClassDisplayProps> = ({ classes }) => {
       <Paper className="p-4 border shadow h-max w-max">
         <Grid gutter="md" justify="center" align="center" grow>
           {classArray.map((cls, index) => (
-            <Grid.Col span={{ base: 12, sm: 6, lg: 4 }} key={cls} className="flex items-center justify-normal">
+            <Grid.Col span={{ base: 6, sm: 6, lg: 4 }} key={cls} className="flex items-center justify-center">
               <Badge color={colors[index % colors.length]} >{cls}</Badge>
             </Grid.Col>
```

Comment:
**Suggestion:** Consider using a constant for the base, sm, and lg values in the `span` prop of `Grid.Col` to avoid hardcoding these values. [best practice]
```suggestion
            const SPAN_VALUES = { base: 6, sm: 6, lg: 4 };
            ...
            <Grid.Col span={SPAN_VALUES} key={cls} className="flex items-center justify-center">
              <Badge color={colors[index % colors.length]} >{cls}</Badge>
            </Grid.Col>
```